coops-ui/tests/unit/Directors.spec.ts

```diff
@@ -598,7 +606,7 @@ describe('Appoint New Director tests', () => {
     setTimeout(() => {
       expect(vm.$el.querySelectorAll('.v-messages')[0].textContent).toBe('')
       done()
-    }, 100)
+    }, 200)
```

Comment:
Just out of curiosity, what is the purpose of the timeout?

Reply:
It's because waiting for the next tick doesn't work in this scenario... unsure why but enough time was wasted on it... so this is a work-around but it works!
Since unit tests appear to be run concurrently, when there are more tests this number sometimes need to be increased.
Honestly, I would like to make this work correctly once and for all, but as I said, this was time-boxed.